massgen/skills/massgen-log-analyzer/SKILL.md

```
@@ -15,18 +15,140 @@ The log-analyzer skill helps you:
+uv run massgen logs analyze --mode self --force         # Overwrite existing report without prompting
+uv run massgen logs analyze --mode self --turn 2        # Analyze specific turn
+uv run massgen logs analyze --mode self --config PATH   # Use custom config
+```
+
+Self-analysis mode runs MassGen with multiple agents to analyze logs from different perspectives (correctness, efficiency, behavior) and produces a combined ANALYSIS_REPORT.md.
+
+### Multi-Turn Sessions
+
+MassGen log directories support multiple turns (coordination sessions). Each turn has its own `turn_N/` directory with attempts inside:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)